src/Event.h

```diff
@@ -85,6 +85,9 @@ enum EventType {
   // Use .syscall.
   EV_SYSCALL,
 
+  // No associated data
+  EV_DETACH,
```

Comment:
Please document this a bit.

Reply:
Why do we need the `EV_DETACH` event instead of just using `EXIT`?

Reply:
I guess, EV_EXIT would be fine. I originally thought I'd want to record more state, but that didn't turn out to be necessary.